apps/crn-server/src/handlers/webhooks/cronjob-sync-orcid.ts

```diff
@@ -35,22 +34,31 @@ const rawHandler = async (): Promise<lambda.Response> => {
   );
   const assetDataProvider = new AssetSquidexDataProvider(userRestClient);
   const users = new Users(userDataProvider, assetDataProvider);
+  const orcidLastSyncDate = DateTime.now()
+    .set({ hour: 0, minute: 0 })
+    .minus({ months: 1 })
+    .toISO();
 
-  const { items: outdatedUsers } = await users.fetch({
-    take: 30,
+  const { items: outdatedUsers } = await userDataProvider.fetch({
+    take: 100,
     filter: {
       orcid: '-',
+      orcidLastSyncDate,
     },
   });
 
+  // orcid rate limit 24 request per second
+  const throttle = pThrottle({
+    limit: 24,
+    interval: 1000,
+  });
```

Comment:
This will likely need to be looked at again once we use contentful data providers since each profile will potentially require several API calls to sync the works, and the contentful management SDK is limited to 7req/s.

It's probably ok for now though.